conda_forge_tick/utils.py

```diff
@@ -182,6 +180,11 @@ def __enter__(self) -> "LazyJson":
     def __exit__(self, *args: Any) -> Any:
         self._dump()
 
+    def __update__(self, data: dict) -> None:
+        self._load()
+        for key in data.items():
+            self.data[f"{key}"] = data[f"{key}"]
```

Comment:
I would use the object's `setitem`: `self[key] = data[key]` that way you don't need to do the load and dump

Reply:
uhm, you are right

Reply:
I would just use the `[]` syntax directly. Also you don't need an f-string for the keys, they are already strings.

Reply:
> I would just use the `[]` syntax directly. Also you don't need an f-string for the keys, they are already strings.

Oh thanks, I was not sure about the syntax usage.